### PR TITLE
#105: added close-to-failed-output tests

### LINKED ISSUE #105: close-to-failed-output.eo:28-31: To add several tests for...
The puzzle `41-17873f95` from #41 has to be resolved: 

https://github.com/objectionary/eo-hamcrest/blob/b27a1fb3a4550ea5dc1880f988a9d4ffe0f977e6/src/test/eo/org/eolang/hamcrest/failed-output/close-to-failed-output.eo#L28-L31

The puzzle was created by graur on 14-Sep-22. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by includealex**
Closes #105.

### CODE REVIEW

File: src/test/eo/org/eolang/hamcrest/failed-output/close-to-failed-output.eo [hunk 1]
```diff
@@ -20,14 +20,35 @@
...
-[] > close-to-failed-output
-  nop > @
+[] > float-is-close-to-float-with-delta-test-failed-output
+  [] > suggestion
+    assert-that > @
+      3.0
+      $.close-to 2.5 0.4
+  assert-that > @
+    suggestion
+    $.equal-to "\nExpected: a numeric value within <2.5> of <0.4>\n     but: <3.0> differed by <0.09999999999999998> more than delta <0.4>"
```
Comment (Graur) [inline src/test/eo/org/eolang/hamcrest/failed-output/close-to-failed-output.eo:36]: @includealex Let's try to use `number.ceil` for this case in https://github.com/objectionary/eo-hamcrest/blob/790e917fa4ad9711f4d3bb33f75f8b6d0cff27fc/src/main/eo/org/eolang/hamcrest/assert-that.eo#L211-L215

**Review by Graur — CHANGES_REQUESTED** [review]
@includealex Good one!

**Comment by includealex** [comment]
@Graur please, check it now.

**Review by Graur — APPROVED** [review]
@includealex Thanks!

**Pull request merged**